tests/conftest.py

```diff
@@ -204,6 +205,25 @@ def shipping_method(shipping_zone):
         price=Money(10, 'USD'), shipping_zone=shipping_zone)
 
 
+@pytest.fixture
+def shipping_zone_weight_based(db):  # pylint: disable=W0613
+    shipping_zone = ShippingZone.objects.create(
+        name='Europe', countries=[code for code, name in countries])
+    shipping_zone.shipping_methods.create(
+        name='DHL', minimum_order_weight=Weight(kg=0),
+        type=ShippingMethodType.WEIGHT_BASED, price=Money(10, 'USD'),
+        shipping_zone=shipping_zone)
+    return shipping_zone
+
+
+@pytest.fixture
+def shipping_method_weight_based(shipping_zone_weight_based):
+    return ShippingMethod.objects.create(
```

Comment:
Do you have anything in mind we could refactor it to? I was trying to have the fixtures independent of each other so that tests don't fail when there are changes in the future.